sbin/build.sh

```diff
@@ -518,13 +518,13 @@ makeACopyOfLibFreeFontForMacOSX() {
 }
 
 
-# Get the first tag from the git repo
+# Get the tags from the git repo and choose the latest tag when there is more than one for the same SHA.
 # Excluding "openj9" tag names as they have other ones for milestones etc. that get in the way
 getFirstTagFromOpenJDKGitRepo()
 {
     git fetch --tags "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}"
-    revList=$(git rev-list --tags --topo-order --max-count=$GIT_TAGS_TO_SEARCH)
-    firstMatchingNameFromRepo=$(git describe --tags $revList | grep jdk | grep -v openj9 | head -1)
+    revList=$(git rev-list --exclude="*_openj9*" --tags="jdk-*" --topo-order --max-count=$GIT_TAGS_TO_SEARCH)
+    firstMatchingNameFromRepo=$(git tag --points-at $revList | grep '+' | sort -V | tail -1)
```

Comment:
What is the `grep '+'` for in here? I don't believe that'll match tags in the JDK8 repo

Reply:
Seems this was from some jdk12 code. Have removed this so it should work with all jdk versions.